Gems/WhiteBox/Code/Source/Rendering/Atom/WhiteBoxAtomRenderMesh.h

```diff
@@ -83,11 +82,13 @@ namespace WhiteBox
         bool DoesMeshRequireFullRebuild(const WhiteBoxMeshAtomData& meshData) const;
 
         AZ::EntityId m_entityId;
+        bool m_visible = true;
```

Comment:
update: It's usually a good idea to put smaller fields at the end of the class definition to help avoid alignment padding - suggest moving this to the end of the fields